algorithms/dqn/utils.py

```diff
@@ -145,7 +145,11 @@ def calculate_c51_loss(
     delta_z = float(v_max - v_min) / (atom_size - 1)
 
     with torch.no_grad():
+        # According to noisynet paper,
+        # Re-sample noisy net parameter on online network when using double q
+        # But, don't implement because performance issue
```

Comment:
, but we don't resample because there is no remarkable difference in performance.